core/controllers/platform_feature.py

```diff
@@ -101,7 +101,7 @@ def get(self):
         # This handler is gated by the dummy_feature flag, i.e. it's only
         # visible when the dummy_feature is enabled.
         if not platform_feature_services.is_feature_enabled(
-                platform_feature_list.PARAM_NAMES.dummy_feature.value):
+                platform_feature_list.PARAM_NAMES.DUMMY_FEATURE.value):
```

Comment:
Shouldn't PARAM_NAMES be UpperCamelCased too?

Reply:
Got you! I checked again It should be good now.